Components/RectangleButton.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { StyleSheet, Text, View, ScrollView, Image, StatusBar, FlatList, TouchableOpacity, TouchableNativeFeedback,
+  Button,
+  Platform } from 'react-native';
+import { DefaultTheme, Headline, Paragraph, TextInput,
+  Appbar, AppbarAction, AppbarBackAction, AppbarContent, AppbarHeader, 
+  Card, CardContent,
+  // Button,
+  ListAccordion, Divider,
+  ListSection, withTheme,
+  Toolbar, ToolbarBackAction, ToolbarContent, ToolbarAction, 
+  FAB, DrawerSection, Colors,
+  Provider as PaperProvider } from 'react-native-paper';
+import { createStackNavigator, DrawerActions } from 'react-navigation';
+import { Constants } from 'expo';
+import Slideshow from 'react-native-slideshow';
+
+// Props: width, color, inner text, function
+export default class RectangleButton extends React.Component {
+    render() {
+        return(
+            <TouchableNativeFeedback
+              onPress={() => this.props.buttonFunction()}
```

Comment:
More typical to do `onPress={this.props.buttonFunction}`. If you don't understand why `onPress={this.props.buttonFunction}` works and `onPress={this.props.buttonFunction()}` doesn't, let me know. I think I can explain it pretty well.

Reply:
Also, you're going to want to press in `this.props.onPress` instead of `buttonFunction`. Trust me on that.